extensions/find-permissions/main.ts

```diff
@@ -241,10 +276,27 @@ function test(directories: string[]): boolean {
 }
 
 // Get the value of a CLI argument.
-function getArgOption(option: string): string | undefined {
-  const option_index = Deno.args.findIndex((arg) => arg === option);
-  if (option_index !== -1) {
-    return Deno.args[option_index + 1];
+//
+// This will skip `offset` number of matching options.
+function getArgOption(option: string, offset?: number): string | undefined {
+  offset = offset ?? 0;
+
+  let optionIndex = -1;
+  for (let i = 0; i < Deno.args.length; i++) {
+    if (Deno.args[i] !== option) {
+      continue;
+    }
+
+    if (offset <= 0) {
+      optionIndex = i;
+      break;
+    }
+
+    offset -= 1;
+  }
```

Comment:
Pretty sure there is a function for this... [`indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)

```suggestion
  const optionIndex = Deno.args.indexOf(option, offset);
```

Reply:
This is subtly different. Here the `offset` is based on array index position, while the offset in my function is based on number of occurrences.